File: Modul4/C/PRAK402-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int angka;
    scanf("%d", &angka);

    for (int i = 1; i < angka+1; i++) {
        if (i % 2 == 1) {
            printf("%d ", i);
        }
    }
    printf("\n");
    for (int j = angka; j > 1; j--) {
        if (j % 2 == 0) {
            printf("%d ", j);
        }
    }
    return 0;
}

File: Modul4/Python/PRAK402-2310817210003-MuhammadAzrianzan.py
angka = int(input(""))

for i in range(1, angka+1):
    if i % 2 == 1:
        print(i, end=" ")

print("")

for j in range(angka, 1, -1):
    if j % 2 == 0:
        print(j, end=" ")